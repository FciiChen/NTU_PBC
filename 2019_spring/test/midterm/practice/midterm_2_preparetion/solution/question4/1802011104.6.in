九合一大選投票時間已在24日下午4點截止，目前開票中。台北市長選情，柯文哲和國民黨丁守中票數呈現拉距，根據TVBS 17點半左右的開票結果，丁約78903、柯約88709。現場支持者也很緊張地在靜候結果。柯文哲幕僚黃瀞瑩表示，對於投票結果真的很緊張，未來有沒有工作就看今夜了。

根據規劃，柯文哲預計在晚間8點時到國際會議中心舉行國際記者會，目前正在休息，由於今年投票狀況多影響開票時間，柯辦表示記者會恐會延後舉行。根據17點30分的開票結果，柯文哲暫時以88709領先丁守中的78903票，姚文智則拿到38808。

柯辦人員表示，今天天台北天氣很好，投票率應該不差，雖然因公投票造成投開票所大排長龍，但柯文哲支持者多數為年輕人體力較好，應該不至於影響投票意願。柯辦也在媒體帳篷內架設12台電視，隨時掌控各家電視台開票結果。（推薦閱讀：花蓮也塞爆！投票未果變返鄉吃飯 張善政：陪丁守中看開票要緊）

學姊黃瀞瑩與海選發言人蔡駿維今日上台主持柯文哲競選晚會，他們對台下民眾表示目前開票作業仍在持續進行，周邊有許多攤販，民眾若肚子餓可以先去享用美食。本身也是台北人的黃瀞瑩表示，上午投票排隊時也排了超過1個小時。
BREAK