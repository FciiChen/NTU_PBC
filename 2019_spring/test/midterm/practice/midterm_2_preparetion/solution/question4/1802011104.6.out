柯_哲 6
8_0 5
票_果 5
7_9 4
丁_中 3
開_結 3
黃_瑩 3
1_點 2
8_7 2
9_3 2
